### ISSUE TITLE
Doc: Step 10 - FRR instead of WireGuard

### ISSUE THREAD

**Issue by bbros-dev** [bmullan/CIAB.Full-Mesh.VPN.Wireguard.FRR.BGP.VXLAN.Internet.Overlay.Architecture#3]
I think the FRR reference is a typo in step 10, where this:

```bash
Example:  Use vwgen to extract each Node’s FRR config file from the Master config file.
```
should be 

```bash
Example:  Use vwgen to extract each Node’s WireGuard config file from the Master config file.
```

**Comment by bmullan** [comment]
Thanks for spotting these things... !

brian

**Comment by bbros-dev** [comment]
Hppy to help.  I'm porting the PDF to markdown and will fix there.  At the moment google: ciab doesn't return this project - but it does bring up your ciab-desktop site.
I don't think it is solely due to the project name, so hopefully MD and github-pages or some such html doc site will help get your ideas out?

**Comment by bbros-dev** [comment]
Closed by RFC #11
